src/libpmemset/source_posix.c

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright 2020, Intel Corporation */
+
+/*
+ * source_posix.c -- implementation of config API (posix)
+ */
+
+#include <string.h>
+
+#include "libpmemset.h"
+
+#include "pmemset_utils.h"
+#include "source.h"
+
+/*
+ * pmemset_source_from_file -- initializes source structure and stores a path
+ *                             to the file
+ */
+int
+pmemset_source_from_file(struct pmemset_source **src, const char *file)
+{
+	LOG(3, "src %p file %s", src, file);
+	PMEMSET_ERR_CLR();
+
```

Comment:
please check pmemset_source for NULL

Reply:
and maybe file param - but i am not sure here

Reply:
I'll check the file param, but pmemset_source isn't checked in *pmemset_source_from_pmem2*, neither is pmem2_source checked in *pmem2_source_from_fd*.